src/community.rs

```diff
@@ -2,26 +2,34 @@ use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, AccountId};
 
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Wiki {
+    name: String,
+    content_markdown: String,
+}
+
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Community {
     pub name: String,
     pub description: String,
     pub image_url: String,
     pub thumbnail_url: String,
-    pub overview_page_markdown: String,
-    pub events_page_markdown: String,
+    pub wiki1: Option<Wiki>,
+    pub wiki2: Option<Wiki>,
     pub admins: Vec<AccountId>,
-    pub labels: Vec<String>,
-    pub telegram_handles: Vec<String>,
+    pub tag: String,
+    pub telegram_handle: Option<String>,
```

Comment:
Do we have any communities in the contract? If so, this is going to be a state-breaking-change

Reply:
AFAIK we don't ( CC @ailisp )

Reply:
No we don't, it's still okay to adjust this struct without worrying about migration